starter_code/views/beers.hbs

```diff
@@ -0,0 +1,6 @@
+<h2>Beers List</h2>
+{{!-- {{#each beersList}}
+<p>{{this.name}}</p>
+{{/each}} --}}
+
+{{> beerPartial}}
```

Comment:
You didn't use the partial exactly how you're meant to. You should loop over your array here and the partial should set the structure for each item - exactly like you did in the version you commented out